### ISSUE TITLE
[Bug][Rating] - NVDA does not read the value when the component is tabbed or the value is changed

### ISSUE THREAD

**Issue by filipKovachev** [telerik/kendo-vue#646]
**Describe the bug**
Currently NVDA does not read the value of the component when being tabbed, value is changed with the mouse or keyboard

**To Reproduce**

1. Open the following example: https://stackblitz.com/edit/vx9fjj?file=src%2Fmain.vue
2. Focus it using tab, change the value with the mouse or keyboard
3. Notice that NVDA does not read the value

**Expected behavior**
NVDA should be reading the value when the component is being tabbed or if the value is changed.

Reported in Ticket ID: 1672153
